lib/iris/analysis/__init__.py

```diff
@@ -1026,18 +1026,35 @@ def _percentile(data, axis, percent, **kwargs):
     If a new additive dimension is formed, then it will always be the last
     dimension of the resulting percentile data payload.
 
+    Kwargs:
+
+    * percentile_method (string) :
```

Comment:
by providing a string input, you are enabling a further enhancement opportunity, specifying another implementation; which is no bad thing, this is just an observation.

Given the string input, I would like to see an enhanced docstring, describing the choice of inputs and their effect. for example

```
Switch to choose between methods for calculating percentiles.
Specify: `percentile_method='fast'` to use the fast `numpy.percentiles` method.
Do not specify or specify `percentile_method=None` to use the scipy.mstats.mquantiles method. 
```